evaluation/welcome_text.md

```diff
@@ -0,0 +1,29 @@
+```
+Dear [NAME],
+
+thank you very much for participating in our small evaluation for the Master Team Project.
+
+You will find detaield instructions concerning your task here: https://d067659.github.io/tpmc_documentation/evaluation/use_case_based_evaluation_process.html.
```

Comment:
Can you make this a hyperlink?

Reply:
But we will send this message via discord/email, I do not think that this will work after copy & paste. We cannot send them a link to this GitHub repo, because they need to know the activation key